examples/jsm/loaders/FBXLoader.js

```diff
@@ -2886,11 +2886,21 @@ var FBXLoader = ( function () {
 			var yIndex = - 1;
 			var zIndex = - 1;
 
+			const indexMap = {}
```

Comment:
You have to change `FBXLoader` in `examples/js` instead and then generate the version in `examples/jsm` with a script. You execute it like so:
```
node utils/modularize.js
```
Besides, please use no ES6 features. So no `const` and no `for/of` loop.